spec/money_spec.rb

```diff
@@ -618,6 +618,16 @@
     end
   end
 
+  describe "calculate_splits" do
+    specify "#calculate_splits gives 1 cent to both people if we start with 2" do
+      number_high, number_low, high, low = Money.new(0.02).calculate_splits(2)
```

Comment:
Can you specify a currency for this money object? We're looking to deprecate instantiating Money objects without currency.

Reply:
Done :)